doc/article/en-US/binding-basics.md

```diff
@@ -25,10 +25,10 @@ Aurelia supports binding HTML and SVG attributes to JavaScript expressions. Attr
 * `attribute`:  an HTML or SVG attribute name.
 * `command`: one of Aurelia's attribute binding commands:
   * `one-time`: flows data one direction: from the view-model to the view, **once**.
-  * `one-way` / `to-view`: flows data one direction: from the view-model to the view.
+  * `to-view`: flows data one direction: from the view-model to the view.
```

Comment:
I think we should leave reference to `one-way` so folks will know that `one-way` and `to-view` are the same thing.  `to-view` should definitely be the preferred command.

Reply:
OK, I wasn't sure if it should be fully removed or not.